src/mumble/VersionCheck.cpp

```diff
@@ -196,3 +224,66 @@ void VersionCheck::fetched(QByteArray a, QUrl url) {
 
 	deleteLater();
 }
+#else // USE_APPIMAGE_UPDATER_BRIDGE
+void VersionCheck::handleUpdateCheck(bool updateAvailable,const QJsonObject &info){
+	(void)info;
+	disconnect(m_Revisioner, &AppImageDeltaRevisioner::updateAvailable,
+		   this, &VersionCheck::handleUpdateCheck);
+	disconnect(m_Revisioner, &AppImageDeltaRevisioner::error,
+		   this, &VersionCheck::handleUpdateCheckError);
+
+	if(!updateAvailable){ // Notify the user they are using the latest version.
+		g.mw->msgBox(QString::fromUtf8("You are currently using the latest version of Mumble AppImage."));
```

Comment:
In Mumble I have been told to prefer `fromLatin1` over `fromUtf8` when er possible as this is how the rest of the code base handles it